### PR TITLE
Update the Reviewers and fixed pre-commit

### PR DESCRIPTION
**Pull request by harshad16**
Update the Reviewers and fixed pre-commit
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

**Comment by harshad16** [comment]
cc: @MichaelClifford 
please review this and check on the list of folks.

File: OWNERS [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-  - pacospace
+  - codificat
+  - fridex
+  - goern
   - harshad16
   - kpostoffice
+  - michaelclifford
+  - TreeinRandomForest
+  - pacospace
+  - sub-mod
```
Comment (MichaelClifford) [inline OWNERS:18]: can sub-mod remain an approver?

Comment (MichaelClifford) [reply OWNERS:18]: @harshad16 ?

Comment (harshad16) [reply OWNERS:18]: yes surely

**Comment by MichaelClifford** [comment]
as Prow is down, I will merge this manually

**Pull request merged**